runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ParDoTranslation.java

```diff
@@ -247,8 +248,15 @@ public static TupleTagList getAdditionalOutputTags(AppliedPTransform<?, ?, ?> ap
       return ((ParDo.MultiOutput<?, ?>) transform).getAdditionalOutputTags();
     }
 
-    RunnerApi.PTransform protoTransform =
-        PTransformTranslation.toProto(application, SdkComponents.create());
+    SdkComponents components = SdkComponents.create();
```

Comment:
Can we at least abstract this away into a static method e.g. `SdkComponents.createDefault(PipelineOptions)`?

Reply:
Makes sense.